projects/cdk/package.json

```diff
@@ -19,7 +19,7 @@
         "@ng-web-apis/mutation-observer": "2.0.0",
         "@ng-web-apis/resize-observer": "2.0.0",
         "@tinkoff/ng-event-plugins": "3.0.0",
-        "@tinkoff/ng-polymorpheus": "4.0.4",
+        "@tinkoff/ng-polymorpheus": "4.0.6",
```

Comment:
please, pin all dependencies with `^`

```
        "@ng-web-apis/mutation-observer": "^2.0.0",
        "@ng-web-apis/resize-observer": "^2.0.0",
        "@tinkoff/ng-event-plugins": "^3.0.0",
        "@tinkoff/ng-polymorpheus": "^4.0.6",
```

Reply:
Why? These are the exact versions we want to supply with `cdk`, I don't think we should add space for change here.